### ISSUE TITLE
Creator search results not getting read back with the screen reader

### ISSUE THREAD

**Issue by srerickson** [psu-libraries/scholarsphere#890]
Should be solved by moving screen reader help text to above the search field.

**Comment by binkylush** [comment]
Screen reader now reads search results
